tests/e2e/e2e_test.go

```diff
@@ -920,13 +926,15 @@ func TestE2E(t *testing.T) {
 							{
 								Name: workerScanName,
 								ComplianceScanSpec: compv1alpha1.ComplianceScanSpec{
-									ContentImage:             contentImagePath,
-									Profile:                  "xccdf_org.ssgproject.content_profile_moderate",
-									Content:                  rhcosContentFile,
-									Rule:                     "xccdf_org.ssgproject.content_rule_no_netrc_files",
-									NodeSelector:             selectWorkers,
+									ContentImage: contentImagePath,
+									Profile:      "xccdf_org.ssgproject.content_profile_moderate",
+									Content:      rhcosContentFile,
+									Rule:         "xccdf_org.ssgproject.content_rule_no_netrc_files",
+									NodeSelector: selectWorkers,
 									ComplianceScanSettings: compv1alpha1.ComplianceScanSettings{
-										RawResultStorageRotation: 1,
+										RawResultStorage: compv1alpha1.RawResltStorageSettings{
```

Comment:
This typo prevents the tests from compiling.

Reply:
ugh... I had fixed it localy and forgot to push it.